### ISSUE TITLE
feat(bridge-ui): improve accessibility

### ISSUE THREAD

**Issue by jscriptcoder** [taikoxyz/taiko-mono#13248]
The idea here is to identify accessibility problems and fix them in order to widen our target audience.

Some of the points to cover:
1. Users should be able to easily read content: font size, color and contrast (Requires design)
2. Users should be able to easily navigate the bridge via keyboard
3. Content should be readable by screen readers: semantic html, aria attributes and roles.

**Comment by jscriptcoder** [comment]
We might want to wait for design to work on this one. Markup will most likely change.

**Comment by d1onys1us** [comment]
@jscriptcoder should we break this out into some subtasks you think?

**Comment by jscriptcoder** [comment]
We can close this one, @d1onys1us , since accessibility will be taken care of as we work on the new UI
